boards/mulle/include/board.h

```diff
@@ -108,54 +108,53 @@ void board_init(void);
  * @todo Work around missing RESET pin on Mulle v0.6x
  *
  * {spi bus, spi speed, cs pin, int pin, reset pin, sleep pin}
+ * @{
  */
-#define AT86RF2XX_PARAMS_BOARD      {.spi = SPI_DEV(0), \
-                                     .spi_clk = SPI_CLK_5MHZ, \
-                                     .cs_pin = SPI_HWCS(1), \
-                                     .int_pin = GPIO_PIN(PORT_B, 9), \
-                                     .sleep_pin = GPIO_PIN(PORT_E, 6), \
-                                     .reset_pin = GPIO_PIN(PORT_C, 12)}
-
+#define AT86RF2XX_PARAM_CS              SPI_HWCS(1)
+#define AT86RF2XX_PARAM_INT             GPIO_PIN(PORT_B, 9)
+#define AT86RF2XX_PARAM_SLEEP           GPIO_PIN(PORT_E, 6)
+#define AT86RF2XX_PARAM_RESET           GPIO_PIN(PORT_C, 12)
+/** @} */
 
 /**
  * @name LIS3DH configuration
  * @{
  */
-#define LIS3DH_INT1                 GPIO_PIN(PORT_C, 18)
-#define LIS3DH_INT2                 GPIO_PIN(PORT_C, 17)
-#define LIS3DH_CS                   SPI_HWCS(0)
-#define LIS3DH_CLK                  SPI_CLK_5MHZ
-#define LIS3DH_SPI                  SPI_DEV(0)
+#define LIS3DH_PARAM_INT1               GPIO_PIN(PORT_C, 18)
+#define LIS3DH_PARAM_INT2               GPIO_PIN(PORT_C, 17)
+#define LIS3DH_PARAM_CS                 SPI_HWCS(0)
+#define LIS3DH_PARAM_CLK                (SPI_CLK_5MHZ)
```

Comment:
just noticed: in all the other places we removed the parenthesis around non-numeric values (e.g. the three lines above). So shouldn't this be the case here, too?

Reply:
I don't think so

Reply:
Regarding those parenthesis, this PR just applies the same 'rule' that is already used with other drivers (at86rf2xx and w5100 at least). I prefer that we keep them with simple defines like this but don't when using a macro function (such as GPIO_PIN, etc). What do you think @haukepetersen ?